src/main/java/org/commcare/modern/database/DatabaseHelper.java

```diff
@@ -34,6 +34,20 @@
     public static Pair<String, String[]> createWhere(String[] fieldNames, Object[] values,
                                                      EncryptedModel em, Persistable p)
             throws IllegalArgumentException {
+        return createWhere(fieldNames, values, null, null, em, p);
+    }
+
+     public static Pair<String, String[]> createWhere(String[] fieldNames, Object[] values,
+                                                     String[] inverseFieldNames, Object[] inverseValues,
+                                                     Persistable p)
+             throws IllegalArgumentException {
+         return createWhere(fieldNames, values, null, null, null, p);
```

Comment:
The inverse values and inverse field names should still be getting passed on here.